examples/golang-http/docker-compose-go.yaml

```diff
@@ -6,7 +6,8 @@ services:
       target: golang-base
     command: >
       bash -c "
-      cd examples/golang/simple
+      cd examples/golang-http/simple
+      && go mod tidy
```

Comment:
if i remove this, the ci would report build errors like this, but it's ok on my local, that is, running `go mod tidy` does not bring up any changes, and running `./verify.sh` or `go build` manually is also no problem.  
![image](https://github.com/envoyproxy/envoy/assets/52443884/b4c9bdc8-259d-42a2-9508-28afa8a94609)

Reply:
iiuc this only needs to be run once - ie when creating the sandbox - its not necessary to run it every time the sandbox runs

the errors you are pointing to are because you removed the `go.sum` files

Reply:
got it, `go.sum` files are ignored by https://github.com/envoyproxy/envoy/pull/24231
```
> git check-ignore -v examples/golang-http/simple/go.sum 
.gitignore:40:*.s??	examples/golang-http/simple/go.sum
```